localization/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -428,13 +429,8 @@ void EKFLocalizer::measurementUpdatePose(const geometry_msgs::msg::PoseWithCovar
   delay_time = std::max(delay_time, 0.0);
 
   int delay_step = std::roundf(delay_time / ekf_dt_);
-  if (delay_step > params_.extend_state_step - 1) {
-    warning_.warnThrottle(
-      fmt::format(
-        "Pose delay exceeds the compensation limit, ignored. delay: %f[s], limit = "
-        "extend_state_step * ekf_dt : %f [s]",
-        delay_time, params_.extend_state_step * ekf_dt_),
-      1000);
+  if (delay_step >= params_.extend_state_step) {
+    warning_.warnThrottle(delayStepWarningMessage(delay_step, params_.extend_state_step), 2000);
```

Comment:
As well as https://github.com/autowarefoundation/autoware.universe/pull/2694#discussion_r1084867985, I think the message should provide pose/twist information.

Reply:
Updated in [880dc3f](https://github.com/autowarefoundation/autoware.universe/pull/2657/commits/880dc3fd335d53238e5d3be497a92a5e7b9eca5c)